dart_io_mini_samples/example/files_directories_and_symlinks/files/handling_errors_when_reading_a_file.dart

```diff
@@ -12,14 +9,11 @@ void handleError(e) {
   print(e.message);
 }
 
-main() {
+main() async {
   final filename = 'non_existent_file.txt';
-  new File(filename).readAsString()
-    // Read and print the file contents.
-    .then(print)
-    // Catch errors.
-    .catchError((e) {
-      print('There was a ${e.runtimeType} error');
-      print('Could not read $filename');
-    });
+  try {
+    var file = await new File(filename).readAsString();
+  } catch(e) {
+    handleError(e);    
```

Comment:
Weird that it didn't use handleError before. Maybe delete handleError(e) here and have the old catchError code here, since it refers to filename?